content/400-reference/200-api-reference/250-error-reference.mdx

```diff
@@ -396,3 +396,48 @@ Possible errors:
 "The schema of the introspected database was inconsistent: {explanation}"
 
 <!-- Error codes generated by https://github.com/mhwelander/doc-tools/blob/master/error_code_parser.py -->
+
+### Prisma Client (Data Proxy)
+
+#### <inlinecode>P5000</inlinecode>
+
+"This request could not be understood by the server"
+
+#### <inlinecode>P5001</inlinecode>
+
+"This request must be retried"
+
+#### <inlinecode>P5002</inlinecode>
+
+The datasource is an invalid one:
+
+* "Could not parse URL of the datasource"
+* "Datasource URL should use prisma:// protocol"
```

Comment:
Is the "protocol" part of this error message a variable? Do we mean to have the space before "protocol"?